charts/test.yaml

```diff
@@ -0,0 +1,8 @@
+# See https://github.com/helm/chart-testing#configuration
```

Comment:
is it really used?

Reply:
it would appear not. It's also not where we're failing though.